index.html

```diff
@@ -17,14 +17,33 @@
             <. / Vi$hal>
         </a>
 
-        <nav class="burger">
+        <div class="burger">
             <i class="fa fa-bars fa-2x" aria-hidden="true"></i>
-        </nav>
+        </div>
+        <nav class="menu-mobile">
+            <ul class="nav-links-mobile">
+              <li>
+                <a href="#Projects" class="a-links">Portfolio</a>
+              </li>
+              <li>
+                <a href="#about_section" class="a-links">About</a>
+              </li>
+              <li>
+                <a href="#contact-me"class="a-links">Contact</a>
+              </li>
+              <li>
+                <div class="close-icon">
+                  <i class="fa fa-times fa-2x" aria-hidden="true"></i>
+                </div>
+              </li>
+            </ul>
+          </nav>
+          
         <nav class="menu">
             <ul>
-                <li><a href="#">Portfolio</a></li>
-                <li><a href="#">About</a></li>
-                <li><a href="#">Contact</a></li>
+                <li><a href="#Projects">Portfolio</a></li>
+                <li><a href="#about_section">About</a></li>
```

Comment:
- [OPTIONAL] I suggest that you make the background text blurry like it is in the given design. This would make your app even closer to the given design and more professional

### Figma design
![](https://user-images.githubusercontent.com/104968236/233144688-e029cd98-5367-432a-ba52-8060f6ad0aaa.png)

### Your design
![](https://user-images.githubusercontent.com/104968236/233144890-1d3eea4b-40b4-4c48-9b66-dddf53a3b3e1.png)